lua/cmp/config/default.lua

```diff
@@ -85,6 +85,8 @@ return function()
       completion = {
         border = { '', '', '', '', '', '', '', '' },
         winhighlight = 'Normal:Pmenu,FloatBorder:Pmenu,CursorLine:PmenuSel,Search:None',
+        col_offset = 0,
+        side_padding = 1,
       },
```

Comment:
I think we also add the `col_offset` and `side_padding` support for `config/window.lua`.

Reply:
I've added the fields and the ability to overwrite them via the table passed in (`opts`). Is there any more to it than that?